ResearchKit/Common/ORKInstructionStepViewController.m

```diff
@@ -60,6 +60,7 @@ - (void)stepDidChange {
         self.stepView.continueSkipContainer.continueButtonItem = self.continueButtonItem;
         self.stepView.headerView.learnMoreButtonItem = self.learnMoreButtonItem;
         self.stepView.continueSkipContainer.continueEnabled = YES;
+        self.stepView.continueSkipContainer.hidden = self.readOnlyMode;
         
         self.stepView.instructionStep = [self instructionStep];
     }
```

Comment:
I think we should always include instruction steps, regardless of embedded or standalone. It helps provide context for the question steps. Though, we should provide a boolean property `excludeInstructionSteps` on the review step with the default value `NO`.